### ISSUE TITLE
It seems that OpenVoice uses VITS's VC to do voice conversion

### ISSUE THREAD

**Issue by MrYANG23** [myshell-ai/OpenVoice#7]

**Comment by Zengyi-Qin** [comment]
Let's use English here so that the discussion can be read by more people. 

1. We would like to emphasize that the contribution of OpenVoice is not inventing the voice converter (which VITS and other works already did), but the **decoupled framework that seperates the voice style and language control from the tone color cloning**.

2. This is **very simple**, but **very effective**, especially when you want to control styles, accents or generalize to new languages. If you wanted to do the same control on XTTS, it could require a lot of data and computing, and no promise to fluently speak any language.

3. Also see our comments in 2.2 Model Structure - Tone Color Converter, in the last two paragraphs. We acknowledged the previous work on flow-based TTS such as VITS and Glow-TTS, and provided insights to the community of why other methods like HuBert-based method could run into some problems. 

4. We also mentioned that the phoneme system can have a huge impact on cross-lingual tone color conversion. This is a insight that VITS and Glow-TTS didn't provide.

In summary, **the contribution of OpenVoice is the Decoupled Framework for Voice Cloning, instead of inventing the individual modules used in the openvoice framework**.

**Comment by Zengyi-Qin** [comment]
At the very beginning, when we started working on instant voice cloning, we realized that if we want to flexibly control the voice styles and languages, we should better seperate **the generation of voice styles** and the **generation of tone color**. This, is what openvoice is all about.

**Comment by MrYANG23** [comment]
Thank you for your reply. Indeed, simplicity and effectiveness are the way to go.
